contracts/lib/tables.cairo

```diff
@@ -1,11 +1,16 @@
 from starkware.cairo.common.registers import get_label_location
 from starkware.cairo.common.math_cmp import is_le
 
-#
-func get_table_encode{range_check_ptr}() -> (encoded_table : felt*):
-    let (table_address) = get_label_location(TABLE_ENCODE)
+func get_char_from_table{range_check_ptr}(index : felt) -> (ascii_value : felt):
+    alloc_locals
```

Comment:
```suggestion
  
```

Can we remove this?

Reply:
Also thanks, I was also looking at such housekeeping changes to optimise code.

Reply:
Just force-pushed an update removing it